### PR TITLE
 Add xclUnmapBO to xrt.h

### PR DESCRIPTION
**Pull request by stsoe**
Call xclUnmapBO when unmapping previously mapped BO.

### CODE REVIEW

File: src/runtime_src/core/include/xrt.h [hunk 1]
```diff
@@ -537,10 +538,19 @@ XCL_DRIVER_DLLESPEC size_t xclReadBO(xclDeviceHandle handle, xclBufferHandle boH
...
+ * To unmap the buffer call xclUnmapBO().
  */
 XCL_DRIVER_DLLESPEC void *xclMapBO(xclDeviceHandle handle, xclBufferHandle boHandle, bool write);
 
+/**
+ * xclUnmapBO() - Unmap a BO that was previously mapped with xclMapBO
+ *
+ * @handle:        Device handle
+ * @boHandle:      BO handle
+ * @addr:          Starting point of mapped address
```
Comment (maxzhen) [inline src/runtime_src/core/include/xrt.h]: it maybe more clear to say: The mapped void * pointer returned from xclMapBO()

**Comment by maxzhen** [comment]
retest this please

**Comment by stsoe** [comment]
retest this please

**Comment by sonals** [comment]
Please update the CHANGELOG

**Comment by stsoe** [comment]
retest this please

**Pull request merged**